docker/docker/src/main/java/org/arquillian/cube/docker/impl/model/DockerCube.java

```diff
@@ -139,10 +155,14 @@ public void stop() throws CubeControlException {
         }
         try {
             lifecycle.fire(new BeforeStop(id));
+
+            long currentTime = System.currentTimeMillis();
             try {
                 executor.stopContainer(id);
             } catch(NotFoundException e) {
             } catch (NotModifiedException e) {}
+            this.stoppingTimeInMillis = System.currentTimeMillis() - currentTime;
+
```

Comment:
Just a thought - can't we collect metrics through events instead of keeping them as a part of the object?